__tests__/mocks/db.json

```diff
@@ -115,5 +115,12 @@
         "primary": true
       }
     ]
+  },
+  "createArticle2": {
+    "title": "share articel",
+    "body": "Quis culpa ad exercitation sunt pariatur ut commodo ipsum ipsum. Labore velit laborum cupidatat laborum excepteur. Lorem eu do cupidatat amet eiusmod veniam.",
+    "description": "exercitation",
+    "status": "published",
+    "tagList": ["test", "article"]
```

Comment:
I suggest having another tag other than test cz someone else is using that which may cause lead to test failure because at the end you are deleting this article basing on this tag and you may end up someone else article

Reply:
it can be done!